upup/pkg/fi/cloudup/awstasks/route.go

```diff
@@ -115,11 +122,14 @@ func (s *Route) CheckChanges(a, e, changes *Route) error {
 		if e.Instance != nil {
 			targetCount++
 		}
+		if e.NatGateway != nil {
+			targetCount++
+		}
 		if targetCount == 0 {
-			return fmt.Errorf("InternetGateway or Instance is required")
+			return fmt.Errorf("InternetGateway | Instance | NatGateway is required")
```

Comment:
Fixed in https://github.com/kubernetes/kops/pull/694/commits/b08794ed173e1f6e3db4daa80a2d703c7d43e0fd